src/Symfony/Component/Security/.gitattributes

```diff
@@ -0,0 +1,5 @@
+Core/Tests/  export-ignore
+Csrf/Tests/  export-ignore
```

Comment:
you use trailing slash here but not in other places like one line below. any reason?

Reply:
also the gitattributes file seems to be missing for the security sub-components

Reply:
the `.gitattributes` and `.gitignore` can also be excluded everywhere

Reply:
Thank you. I missed the security sub-packages. 
I also added .gitattributes and .gitignore.